src/qml/qgismobileapp.qml

```diff
@@ -375,106 +379,126 @@ ApplicationWindow {
     }
   }
 
-  Button {
-    id: gpsButton
-    state: positionSource.active ? "On" : "Off"
-    visible: positionSource.valid
-    round: true
-
+  Column {
+    id: mainToolBar
     anchors.left: dashBoard.right
     anchors.leftMargin: 4 * dp
     anchors.top: mainMenuBar.bottom
     anchors.topMargin: 4 * dp
+    spacing: 4*dp
```

Comment:
Would be nice to stick to the style of the lines above. Too bad there's no astyle for qml (yet?)